java/server/src/main/java/com/squareup/subzero/server/resources/GenerateQrCodeResource.java

```diff
@@ -140,6 +142,15 @@ public QrCode signTxRequest(SignTxRequest request) throws WriterException {
     builder.setWalletId(request.wallet);
     builder.setSignTx(signTxBuilder);
 
+    byte [] command_bytes = builder.build().toByteArray();
+    //Sign with a know dev key. This is for testing only.
+    QrSigner signer = new QrSigner(Hex.decode("3d9b97530af1d91e1d818c9498d8a53d9b97530af1d91e1d818c9498d8a59fe3"));
```

Comment:
It will probably be helpful to also leave a comment here to mention that this value needs to in sync with that in `core/include/config.h` (and leave a comment in `config.h` to refer back here).